react/features/base/label/components/AbstractCircularLabel.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import { Component } from 'react';
+
+export type Props = {
+
+    /**
+     * String that will be rendered as the label itself.
+     */
+    label: string
+};
```

Comment:
You know I don't like to implement things that are not used ;)

Reply:
But it already is. On native. You just have a default value for it.

Reply:
I don't really get what you mean. In order to make the size adjustable (which is not needed atm) we need to extract the size and border setting into a dynamic style object generated run-time (on render, I guess), and that would be a code that we wouldn't use because we'd only go with the default size for now. Or did I misunderstand you?

Reply:
No, that's what I meant, yes. 🤷‍♂️